fillHists.sh

```diff
@@ -16,13 +16,22 @@ if [ "$modTime" -gt "$runTime" ]; then
     echo
 fi
 
-EXE=./Utilities/scripts/makeHistFile.py
-ANA=ThreeLep
-YEAR=2016
-INP=TwoLep_Met40
-# SEL=FourTopCutBasedEl
-SEL=FourTopMVAEl
+# EXE=./Utilities/scripts/makeHistFile.py
```

Comment:
Move this file to the scripts folder

Reply:
that's my personal script I accidentally added to the master, I'll just remove it!

Reply:
If you want you can keep it in scripts, or move it to Scratch (where I put the scripts I want to keep track of that are quick dirty)